src/SourceBuild/content/eng/tools/BinaryToolKit/Modes.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace BinaryToolKit;
+
+public enum Modes
+{
+    both,
+    validate,
+    clean
```

Comment:
This should be defined as you would naturally define an enum, not compensating with the casing for the use of System.CommandLine. There are mappings that can be done in the use of System.CommandLine to get this working. But I think that's overkill anyway. Let's just define the options as `Validate`, `Clean`, and `All` (a more standard option name vs Both). I see no reason to handle these as lowercase for the CLI.